recipe/meta.yaml

```diff
@@ -29,7 +29,7 @@ requirements:
     - configobj >=5.0.6
     - distro >=1.3
     - dpath <3,>=2.1.0
-    - dvc-data >=1.11.0,<1.12.0
+    - dvc-data >=2.3.0,<2.4.0
```

Comment:
Need to wait for `dvc-data>=2.3.0` to be packaged at https://github.com/conda-forge/dvc-data-feedstock.